### ISSUE TITLE
Files missing in the installed package

### ISSUE THREAD

**Issue by warenlg** [src-d/lookout-sdk-ml#46]
Similarly to https://github.com/src-d/style-analyzer/issues/528. many files are missing in the real package. Among them `Dockerfile.jinja2`

Indeed, when running,
`analyzer package lookout.style.format -y -u warenlg -t token -r warenlg/node -w /tmp/pkg`
we met the following error when generating the Dockerfile,
```Traceback (most recent call last):
  File "/usr/local/bin/analyzer", line 11, in <module>
    sys.exit(main())
  File "/home/waren/.local/lib/python3.6/site-packages/lookout/__main__.py", line 18, in main
    return handler(args)
  File "/home/waren/.local/lib/python3.6/site-packages/lookout/core/package.py", line 28, in package_cmdline_entry
    args.repo, args.user, args.token)
  File "/home/waren/.local/lib/python3.6/site-packages/lookout/core/package.py", line 60, in package
    " ".join(analyzers), ndeps, ndeps_dev, package_name, wd, repo, user, token, log)
  File "/home/waren/.local/lib/python3.6/site-packages/lookout/core/package.py", line 153, in _generate_configs
    template = loader.load(jenv, "Dockerfile.jinja2")
  File "/home/waren/.local/lib/python3.6/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/waren/.local/lib/python3.6/site-packages/jinja2/loaders.py", line 187, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: Dockerfile.jinja2
```

**Comment by vmarkovtsev** [comment]
I bet that @carlosms had the same problem. I will fix it asap.

**Comment by vmarkovtsev** [comment]
The fix is in https://github.com/src-d/lookout-sdk-ml/pull/47

**Comment by vmarkovtsev** [comment]
Released in 0.8.1
